[2024-02-28T13:14:02.954+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T13:13:59.528429+00:00 [queued]>
[2024-02-28T13:14:02.966+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T13:13:59.528429+00:00 [queued]>
[2024-02-28T13:14:02.967+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-28T13:14:03.001+0000] {taskinstance.py:2191} INFO - Executing <Task(GCSObjectExistenceSensor): gcs_object_exists> on 2024-02-28 13:13:59.528429+00:00
[2024-02-28T13:14:03.031+0000] {standard_task_runner.py:60} INFO - Started process 78 to run task
[2024-02-28T13:14:03.036+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_snowflake_etl', 'gcs_object_exists', 'manual__2024-02-28T13:13:59.528429+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/GCS_to_Snowflake_DAG.py', '--cfg-path', '/tmp/tmp_vinyb9n']
[2024-02-28T13:14:03.040+0000] {standard_task_runner.py:88} INFO - Job 87: Subtask gcs_object_exists
[2024-02-28T13:14:03.180+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T13:13:59.528429+00:00 [running]> on host b9a0bf70b495
[2024-02-28T13:14:03.317+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='gcs_snowflake_etl' AIRFLOW_CTX_TASK_ID='gcs_object_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T13:13:59.528429+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-28T13:13:59.528429+00:00'
[2024-02-28T13:14:03.320+0000] {gcs.py:98} INFO - Sensor checks existence of : us_stock_market, US-Stock-Market-Dataset.csv
[2024-02-28T13:14:03.334+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-02-28T13:14:03.343+0000] {base.py:83} INFO - Using connection ID 'gcp_conn_id' for task execution.
[2024-02-28T13:14:03.869+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-02-28T13:14:03.884+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_snowflake_etl, task_id=gcs_object_exists, execution_date=20240228T131359, start_date=20240228T131402, end_date=20240228T131403
[2024-02-28T13:14:03.975+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T13:14:04.060+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
